test/core/test_component.py

```diff
@@ -153,35 +153,6 @@ def compare_vals(x, y, z):
     assert baz01 == baz02
 
 
-def test_dynamic_loading():
```

Comment:
Why is this removed? This is not moved anywhere, right? Anything else is moved to a specific `mx` test folder. Why is this not?

Reply:
Good spot! I've removed it since this also only seems to test `load_code` which is also tested elsewhere. I can add it to the mxnet tests though.

Reply:
Not sure what the purpose is but I would keep it, if possible.